docker/src/main/java/clocker/docker/entity/container/application/VanillaDockerApplicationSshDriver.java

```diff
@@ -63,6 +65,16 @@ public void install() {
     }
 
     @Override
+    public void copyRuntimeResources() {
+        Map<String, String> runtimeFiles = entity.getConfig(VanillaSoftwareProcess.RUNTIME_FILES);
+        Map<String, String> runtimeTemplates = entity.getConfig(VanillaSoftwareProcess.RUNTIME_TEMPLATES);
+        if ((runtimeFiles != null && runtimeFiles.size() > 0) ||
+                (runtimeTemplates != null && runtimeTemplates.size() > 0)) {
+            super.copyRuntimeResources();
```

Comment:
Yes, we should, but I didn't want to have to wait to get the change into Brooklyn.